package.json

```diff
@@ -62,6 +62,7 @@
     "flow-bin": "^0.129.0",
     "husky": "^4.2.5",
     "istanbul": "^0.4.5",
+    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#FEC-10057",
```

Comment:
is this correct with FEC-10057?

Reply:
it's just to pass travis. anyway it's peer-dependency so it's still correct